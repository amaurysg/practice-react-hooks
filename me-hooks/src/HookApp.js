import React from 'react'
import CallBackHook from './components/memos/CallBackHook'
import MemoHooks from './components/memos/MemoHooks'
import Memorized from './components/memos/Memorized'
import Layout from './components/useLayoutEffect/RealLayoutEffect'
import TodoApp from './components/useReducer/TodoApp'
// import Characters from './components/examples/Characters'
// import MultipleCustomsHooks from './components/examples/MultipleCustomsHooks'
// import Focus from './components/useRef/Focus'
import RealExampleUseRef from './components/useRef/RealExampleUseRef'
/* import FormCustomsHooks from './components/useEffect/FormCustomsHooks'
import Message from './components/useEffect/Message'
import SimpleForm from './components/useEffect/SimpleForm'
import CounterApp from './components/useState/CounterApp'
import CounterCustomsHooks from './components/useState/CounterCustomsHooks' */
// import Padre from './components/memos/07-tarea-memo/Padre'
// import IntroReducer from './components/useReducer/introReducer'

const HookApp = () => {
  return (
    <div>
{/*       <CounterApp/>
      <hr/>
      <CounterCustomsHooks/>
      <hr/>
      <SimpleForm/>
      <hr/>
      <FormCustomsHooks/> */}

      {/* <MultipleCustomsHooks/>
      <hr/>
      <Characters/> */}
      {/* <hr/>
      <Focus/> */}

      {/* <RealExampleUseRef/> */}
      {/* <Layout/> */}
      {/* <Memorized /> */}
      {/* <MemoHooks /> */}
      {/* <CallBackHook /> */}
      {/* <Padre /> */}
      
      {/* <IntroReducer /> */}

      <TodoApp/>

    </div>
  )
}

export default HookApp
