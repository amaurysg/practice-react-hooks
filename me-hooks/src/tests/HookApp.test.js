import { shallow } from "enzyme";
import HookApp from "../HookApp";

describe("Test to Component HookApp", () => {
  test("Snapshot Render component", () => {
    const wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
