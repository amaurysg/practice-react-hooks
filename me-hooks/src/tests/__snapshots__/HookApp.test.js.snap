// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test to Component HookApp Snapshot Render component 1`] = `
<div>
  <MainApp />
</div>
`;
